src/com/google/enterprise/adaptor/SensitiveValueCodec.java

```diff
@@ -298,15 +310,33 @@ public static void main(String[] args) throws IOException {
       }
     }
     SensitiveValueCodec codec = new SensitiveValueCodec(keyPair);
-    
-    Console console = System.console();
-    if (console == null) {
-      log.warning("Couldn't get Console instance");
-      return;
+    boolean stdinParameterPresent = isParameterPresent(args, "--stdin");
```

Comment:
The console typically is stdin.

So the presence of this flag does not uniquely denote stdin and further the absence of it does not communicate the absence of stdin.

I would prefer "--noconsole" (with --console being OK and same as no argument) or "--quiet".  I think the --stdin is a bit misleading because of the above reasons.

Thank you